modules/gittar/pkg/gitmodule/repo_branch.go

```diff
@@ -163,22 +163,22 @@ func (repo *Repository) DeleteBranch(name string) error {
 }
 
 // CreateBranch create a branch from repository.
-func (repo *Repository) CreateBranch(name string, ref string) error {
+func (repo *Repository) CreateBranch(name string, ref string) (*Commit, error) {
```

Comment:
test empty repository created by `git init`.

Reply:
create new branch to empty repository will return error